docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: docs
+title: Interact With The Page
+permalink: /documentation/guides/basic-guides/interact-with-the-page.html
+---
+# Interact With The Page
+
+Test API provides a set of **actions** that enable you to interact with the webpage.
+
+They are implemented as methods in the [test controller](../../reference/test-api/testcontroller/README.md) object. You can call them in a chained fashion.
```

Comment:
"a chained fashion"? Explain

Reply:
@dirk-pieterse  a concept of [method chaining](https://en.wikipedia.org/wiki/Method_chaining) is referenced here.